### DISCUSSION TITLE
No Mail for password reset

### DISCUSSION THREAD

**Discussion by Dany1410** [kimai/kimai#3359] [General]
Hello,

I just wanted to test the password reset for my Admin user after installing according to the recommended setup.
Unfortunality, I do not get the email with the link to reset my password. 
My .env file looks like this:
![konfig  env error](https://user-images.githubusercontent.com/107126760/173599790-da8eb52b-60fa-4b25-ace8-1888ce33327d.PNG)

Do I have to make any additional changes?
Thank you in advance.

**Comment by kevinpapst** [comment]
Is it too much to ask for to read the documentation that is linked right in front of you?
